Taylor_series_Repo/Taylor_series_Horner_rule.c

```diff
@@ -0,0 +1,59 @@
+/*Description of Algorithm:-

```

Comment:
As a suggestion, you can create a readme for Taylor series using Horner's rule in a separate PR by updating your description which you have here slightly.

If you wish to do so, remove the description from here. It seems pretty big to me for any brief overview. 3-4 lines are enough as an overview.